src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -828,6 +834,53 @@ public void testSubmitLink() {
         feedbackPage = getFeedbackPageForInstructor(idOfInstructorWithSessions);
     }
     
+    public void testCopySessionModalButtons() {
+        feedbackPage.copyFeedbackSessionTestButtons("Session 1", newSession.courseId);
+        
+        assertFalse(feedbackPage.isCopySubmitButtonEnabled());
+        
+        ______TS("click on a row");
+        
+        feedbackPage.clickCopyTableAtRow(0);
+        assertTrue(feedbackPage.isRowSelected(0));
+        assertTrue(feedbackPage.isRadioButtonChecked(0));
+        assertTrue(feedbackPage.isCopySubmitButtonEnabled());
```

Comment:
Would be good to add a "click on another row" here, currently -

row -> radio
radio -> radio
radio -> row

is being tested but row -> row is missing.